api/views/mixins.py

```diff
@@ -0,0 +1,24 @@
+from rest_framework import request, serializers
+from rest_framework.permissions import SAFE_METHODS
+
+
+class ReadWriteSerializersMixin:
+    """
+    A view mixin that provides a way to specify different serializers for
```

Comment:
same (summary line)

Reply:
But don't agree here. I think this comment isn't really "multiline" - it is a summary, but because we have max line length restriction, it's split into multiple lines

Reply:
Hehe :) I guess PEP means that this line should be one line within a limit of one line :)
So we could add one more short line on top.
But this is not really that necessary, of course.

Reply:
Just rephrased it to make shorter and fit into one line :)